modules/ROOT/partials/integrations/react-quick-start.adoc

```diff
@@ -517,6 +474,7 @@ import './tinymce/plugins/directionality/plugin.min.js';
 import './tinymce/plugins/emoticons/plugin.min.js';
 import './tinymce/plugins/fullscreen/plugin.min.js';
 import './tinymce/plugins/help/plugin.min.js';
+import './tinymce/plugins/help/js/i18n/keynav/en.js';
 import './tinymce/plugins/image/plugin.min.js';
 import './tinymce/plugins/importcss/plugin.min.js';
 import './tinymce/plugins/insertdatetime/plugin.min.js';
```

Comment:
Do we need to include `/plugin.min.js` here still in the imports

Reply:
As discussed @kemister85, since the index files won't be added until 7.1, I'll leave that change for #3224, and leave these plugin.min.js imports as is in this PR.